.travis.yml

```diff
@@ -7,6 +7,16 @@ services:
 # a lightweight environment
 language: c
 
+env:
+    - DC=dmd1 DIST=trusty F=production
+    - DC=dmd1 DIST=trusty F=devel
+    - DC=dmd1 DIST=xenial F=production
+    - DC=dmd1 DIST=xenial F=devel
+    - DC=dmd-transitional DIST=trusty F=production
+    - DC=dmd-transitional DIST=trusty F=devel
+    - DC=dmd-transitional DIST=xenial F=production
+    - DC=dmd-transitional DIST=xenial F=devel
```

Comment:
One thing I wonder about is if would be better to use specific versions (`1.080.0`, `2.070.2.s10`) in matrix, so that it can be naturally extended with more.